src/telemetry/index.ts

```diff
@@ -1530,4 +1530,9 @@ export interface IEventNamePropertyMapping {
         ename: string;
         evalue: string;
     };
+    [Telemetry.RunTest]: {
+        testName: string;
+        testResult: string;
+        isCIServer: boolean;
```

Comment:
Why do we need this? 
In production we'd never send such telemetry as users will never have the tests and that code path will never get executed.

Reply:
when we execute tests locally it won't be on a CI server, and I definitely want to be able to filter those out (maybe even not send telemetry for local runs) since they are much more likely to fail when used to try out a code change or new feature

Reply:
Aaah got it, in that case can we add a condition to the hooks to disable sending telemery for locally run tests.
Else currently any test run locally will always send the telemetry